### PR TITLE
Add wizard file

### PR DESCRIPTION
**Pull request by medou-boushab**

### CODE REVIEW

**Review by fviale — APPROVED** [review]

File: app/templates_versions/community/subviews.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
             "workflow-execution/templates/job-results-dropdown.html",
             "workflow-execution/templates/job-service-dropdown.html",
             "workflow-execution/templates/job-signals-dropdown.html",
-            "workflow-execution/templates/workflow-variables.html"
+            "workflow-execution/templates/workflow-variables.html",
+            "workflow-execution/templates/wizard.html"
```
Comment (amouhoub) [inline app/templates_versions/community/subviews.json:18]: Don't merge this part yet as Fabien requested so please put it back like it was

File: app/templates_versions/enterprise/subviews.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
             "workflow-execution/templates/job-results-dropdown.html",
             "workflow-execution/templates/job-service-dropdown.html",
             "workflow-execution/templates/job-signals-dropdown.html",
-            "workflow-execution/templates/workflow-variables.html"
+            "workflow-execution/templates/workflow-variables.html",
```
Comment (amouhoub) [inline app/templates_versions/enterprise/subviews.json:17]: Same as above

**Review by amouhoub — CHANGES_REQUESTED** [review]

File: app/templates_versions/enterprise/subviews.json (hunk 2 above)
Comment (medou-boushab) [reply app/templates_versions/enterprise/subviews.json:17]: I can do it but it won't cause a problem for the build

Comment (medou-boushab) [reply app/templates_versions/enterprise/subviews.json:17]: @mboussaa do you confirm?

Comment (mboussaa) [reply app/templates_versions/enterprise/subviews.json:17]: another PR as suggested

**Pull request merged**